dev/plugins/hu.elte.txtuml.xtxtuml/src/hu/elte/txtuml/xtxtuml/formatting2/XtxtUMLFormatter.xtend

```diff
@@ -137,8 +138,14 @@ class XtxtUMLFormatter extends XbaseFormatter {
 		}
 	}
 
+	def dispatch void format(TUModifiers it, extension IFormattableDocument document) {
+		regionFor.feature(TU_MODIFIERS__STATIC).append[oneSpace];
+		regionFor.feature(TU_MODIFIERS__EXTERNALITY).append[oneSpace];
+		regionFor.feature(TU_MODIFIERS__VISIBILITY).append[oneSpace];	
+	}
+	
 	def dispatch void format(TUConstructor it, extension IFormattableDocument document) {
-		regionFor.feature(TU_CONSTRUCTOR__VISIBILITY).append[oneSpace];
```

Comment:
I recommend to prepend the `name` feature with one space here. Of course this rule will be redundant (see the previous comment), but I believe this is more in line with the aforementioned invariant.